### ISSUE TITLE
[ts] constructor auto-doc doesn't support typed parameters

### ISSUE THREAD

**Issue by facelesspanda** [joelday/vscode-docthis#60]
Here is a simple TS class:
```typescript
export class Generic<T> {
    constructor(readonly _id: number) { }
    someMethod(v : T) { }
}
```
When doing *Document this* on `someMethod`, here's what gets generated :
```typescript
    /**
     * 
     * 
     * @param {T} v
     * 
     * @memberOf Generic
     */
    someMethod(v : T) { }
```
Type of param `v` is `T`, as expected.

But doing *Document this* on `constructor` generates this:
```typescript
    /**
     * Creates an instance of Generic.
     * 
     * @param {any} [readonly=_id]
     * @param {any} number
     * 
     * @memberOf Generic
     */
    constructor(readonly _id: number) { }
```
which creates 2 parameter entries. One for the parameter-initializer with its access modifier, and the other for its type.

**Comment by joelday** [comment]
The language service needs to be updated to TypeScript 2.0 so that we can properly deal with the readonly keyword. Sorry about this, will try to get this fixed up soon.

**Comment by TheOpenDevProject** [comment]
Updating now.

**Comment by joelday** [comment]
@facelesspanda Fixed in 0.3.9. Sorry for the huge delay!
